tutorials/tf2onnx_custom_ops_tutorial.ipynb

```diff
@@ -69,10 +69,10 @@
    "source": [
     "class Model1(tf.keras.Model):\n",
     "\n",
-    "    def __init__(self, name='model1', **kwargs):\n",
```

Comment:
It's better to separate this change into another PR.

Reply:
Oh, those parts of code are changed by my IDE in a mistake, I will revert it.